synapseutils/describe.py

```diff
@@ -0,0 +1,58 @@
+import synapseclient
+import pandas as pd
+
+
+def _open_entity_as_df(syn, entity: str) -> pd.DataFrame:
+    """
+    :param syn: synapse object
+    :param entity: a synapse entity to be extracted and converted into a dataframe
+    :return: a pandas DataFrame if flow of execution is successful; None if not.
+    """
+
+    dataset = None
+
+    try:
+        entity = syn.get(entity)
+        format = entity.path.split(".")[-1]
```

Comment:
Not important, the way you're doing it is totally fine.

```
name, format = os.path.splitext('/path/to/somefile.ext')
```

Reply:
This is easier to read, I added it in

Reply:
This is so much cleaner.  I'm used to the -1 syntax, but this makes it more obvious.